tests/test.py

```diff
@@ -114,20 +138,32 @@ def test_session_stats_create_with_session_record(self):
         self.assertIsNotNone(session.client_updated_at)
 
     def test_session_stats_create_with_negative_values(self):
-        re = self._post('/v1/sessions/123/stats', {
-            'bytes_sent': -20,
-            'bytes_received': 40
-        })
+        signature, public_address = helpers.sign_message_with_static_key('')
+        headers = {
+            "Authorization": "Signature {}".format(base64.b64encode(signature))
```

Comment:
This header construction is heavily duplicated - maybe it's worth to extract it to `tests/helpers.py` ?

Reply:
moved header construction to utils